resources/assets/lib/user-multiplayer-index/room.tsx

```diff
@@ -36,21 +35,8 @@ export default class Room extends React.Component<Props> {
     return diff < endingSoonDiffMs ? 'soon' : 'active';
   }
 
-  @computed
-  private get maxDifficulty() {
-    const max = maxBy(this.props.room.playlist, (playlist) => this.props.store.beatmaps.get(playlist.beatmap_id)?.difficulty_rating);
-    return this.props.store.beatmaps.get(max?.beatmap_id ?? 0)?.difficulty_rating ?? 0;
-  }
-
-  @computed
-  private get minDifficulty() {
-    const min = minBy(this.props.room.playlist, (playlist) => this.props.store.beatmaps.get(playlist.beatmap_id)?.difficulty_rating);
-    return this.props.store.beatmaps.get(min?.beatmap_id ?? 0)?.difficulty_rating ?? 0;
-  }
-
   private get background() {
-    const beatmap = this.props.store.beatmaps.get(this.props.room.playlist[0].beatmap_id);
-    const beatmapset = this.props.store.beatmapsets.get(beatmap?.beatmapset_id ?? 0);
+    const beatmapset = this.props.room.current_playlist_item.beatmap?.beatmapset;
```

Comment:
not really an issue but this can return directly by continuing with `beatmapset?.covers.cover`